crates/aptos/src/op/key.rs

```diff
@@ -167,7 +167,7 @@ impl GenerateKey {
         let args = format!(
             "generate --key-type {key_type:?} --output-file {key_file} --encoding {encoding:?} --assume-yes",
             key_type = KeyType::Ed25519,
-            key_file = key_file.to_str().unwrap(),
+            key_file = key_file.display().to_string(),
```

Comment:
doesn't display return a string?

Reply:
Actually no, for some reason it just changes it to an intermediary type